app/src/main/java/com/mobileasone/dagger2workshop/di/ActivityBuilder.kt

```diff
@@ -0,0 +1,21 @@
+package com.mobileasone.dagger2workshop.di
+
+import com.mobileasone.dagger2workshop.presentation.addnotes.AddNoteActivity
+import com.mobileasone.dagger2workshop.presentation.notedetail.DetailNoteActivity
+import com.mobileasone.dagger2workshop.presentation.notelist.NoteListActivity
+import dagger.Module
+import dagger.android.ContributesAndroidInjector
+
+@Module
+abstract class ActivityBuilder {
+
+    @ContributesAndroidInjector
+    internal abstract fun bindNoteListActivity(): NoteListActivity
```

Comment:
These abstract methods are used to inject stuff through AndroidInjection.inject() methods used in activities.